src/multivectors.jl

```diff
@@ -54,6 +54,7 @@ setindex!(m::Chain{V,G,T} where {V,G},k::T,i::Int) where T = (m.v[i] = k)
 Base.firstindex(m::Chain) = 1
 @pure Base.lastindex(m::Chain{V,G}) where {V,G} = binomial(ndims(V),G)
 @pure Base.length(m::Chain{V,G}) where {V,G} = binomial(ndims(V),G)
+Base.zero(::Type{<:Chain{V,G,T}}) where {V,G,T} = Chain{V,G}(zeros(svec(ndims(V),G,T)))
```

Comment:
is `::Type{<:Chain{V,G,T}}` better than `::Type{Chain{V,G,T}}` just wondering

Reply:
The second version expects that `Chain` has exactly three arguments. Since it's a computed type, it can have a fourth argument. The first version accepts this as well and ignores it.

For example, your suggestion would not work with `zero(typeof(v1+v2))`.